chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/DeveloperManager.java

```diff
@@ -17,6 +18,10 @@
 import gov.healthit.chpl.exception.ValidationException;
 
 public interface DeveloperManager {
+    String NEW_DEVELOPER_CODE = "XXXX";
+    String SYSTEM_DEV_VALIDATION_ERROR = "Unable to validate system developer as the pending ACB Name is null or its String representation "
+            + "is null or empty";
+
```

Comment:
This error message should probably be in the errors.properties.

Reply:
Thanks, good point, updated accordingly